katalog/istio-operator/MAINTENANCE.md

```diff
@@ -28,6 +28,20 @@ All the following examples are tested in the pipeline
 [mtls-test-2](../../katalog/tests/istio-operator/istio-citadel-authn-2.sh)
 [mtls-test-3](../../katalog/tests/istio-operator/istio-citadel-authn-3.sh)
 
+### Dashboard and Rules
+
+#### Grafana Dashboards
```

Comment:
we need to leave an empty line after a header and before and after a list of items, otherwise the linter gets sad :(

```suggestion
#### Grafana Dashboards

```